src/main/java/online/kingdomkeys/kingdomkeys/client/gui/synthesis/SynthesisCreateScreen.java

```diff
@@ -208,9 +208,9 @@ public void render(@NotNull GuiGraphics gui, int mouseX, int mouseY, float parti
 
 			create.active = enoughMats && enoughMunny && enoughTier && enoughSpace;
 			if(!enoughSpace) { //TODO somehow make this detect in singleplayer the inventory changes
-				create.setMessage(Component.translatable("Not enough space"));
+				create.setMessage(Component.translatable(Utils.translateToLocal(Strings.Gui_Synthesis_Synthesise_Create));
 			} else {
-				create.setMessage(Component.translatable(Utils.translateToLocal(Strings.Gui_Synthesis_Synthesise_Create)));
+				create.setMessage(Component.translatable(Utils.translateToLocal(Gui_Shop_NoSpace)));
```

Comment:
Strings.Gui_Shop_NoSpace

Reply:
I'm not sure, but isn't it here a bit redundant to have `Utils.translateToLocal` inside of `Component.translatable` ? Because at other times it does work fine, and at others not.
Or is that required for the `setMessage` function?

Reply:
Yeah you're right translateToLocal just uses Component.translatable anyway #blameabel lol